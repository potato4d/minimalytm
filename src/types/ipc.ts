export enum IPCEventNames {
  PLAY_PAUSE = 'play/pause',
  PREV = 'prev',
  NEXT = 'next'
}
