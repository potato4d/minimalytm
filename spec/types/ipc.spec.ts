import { IPCEventNames } from '../../src/types/ipc'

describe('ipc', () => {
  test('match IPCEventNames', () => {
    expect(IPCEventNames).toMatchSnapshot()
  })
})
