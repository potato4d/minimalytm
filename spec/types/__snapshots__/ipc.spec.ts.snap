// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ipc match IPCEventNames 1`] = `
Object {
  "NEXT": "next",
  "PLAY_PAUSE": "play/pause",
  "PREV": "prev",
}
`;
